src/writing-policies/spec/06-signature-verifier-policies.md

```diff
@@ -0,0 +1,147 @@
+# Signature verifier policies
+
+Kubewarden implements support for the [Sigstore](https://www.sigstore.dev/)
+project. This allows to implement a Secure Supply Chain for your cluster.
+
+Part of that is to ensure that all container images running in the cluster are
+signed and verified, proving that they come from their stated authors, and
+haven't been tampered with. For further reading, do check out the docs on [how we implement a Secure Supply Chain for the policies themselves](TODO link to https://github.com/kubewarden/docs/pull/100/files)).
```

Comment:
TODO missing link to sigstore docs

Reply:
Done in https://github.com/kubewarden/docs/pull/109/commits/61f35a09feffb32c79ada0703fc005184e656c48.